content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -166,6 +166,46 @@ Cancel`.
 
 .. _india/verify-e-invoice:
```

Comment:
please move this custom anchor before the "GST e-Invoice verification" heading and add this one here instead:

```suggestion
.. _india/e-invoice-negative-lines:
```

Thanks!

Reply:
Done